.changes/unreleased/tinymce-TINY-10916-2024-05-15.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Fixed a bug where applying a transparent color resulted in it being converted to black.
```

Comment:
It converts existing colors as well so just loading up a document with `background: transparent` will change the color.